.github/workflows/check-odo.yml

```diff
@@ -0,0 +1,43 @@
+name: check-odo
+
+on: [workflow_dispatch]
+
+jobs:
+  check-odo-repo:
+    runs-on: ubuntu-latest
+    env:
+        TOOL_REPO: redhat-developer/odo
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Check Out Code
+        uses: actions/checkout@v2
+      - name: Get latest ODO version
+        run: |
+          echo "REPO_ODO_VERSION=$(cat src/tools.json | jq -r .odo.version)" >> $GITHUB_ENV
+          LATEST_TOOL_RELEASE_RESP=$(gh release --repo ${{ env.TOOL_REPO }} view --json tagName,url)
+          echo "LATEST_TOOL_RELEASE=$(echo ${LATEST_TOOL_RELEASE_RESP} | jq -r .tagName | sed 's|v||')" >> $GITHUB_ENV
+          echo "LATEST_TOOL_URL=$(echo ${LATEST_TOOL_RELEASE_RESP} | jq -r .url)" >> $GITHUB_ENV
+      - name: Find existing PR for ODO version
+        run: |
+          echo PR_EXISTS=$(gh pr --repo rgrunber/vscode-openshift-tools list --state all --search "update odo ${{env.LATEST_TOOL_RELEASE}} in:title" --json url | jq length) >> $GITHUB_ENV
```

Comment:
`rgrunber/vscode-openshift-tools` I hope wrongly pointed to your repo

Reply:
Yup, I must have accidentally let that go through when testing. With `in:body`, I was worried there might be a risk where a mention of those terms gets interpreted as an existing PR where it shouldn't. Looking back into past PR, https://github.com/redhat-developer/vscode-openshift-tools/pulls?q=is%3Apr+in%3Atitle+odo+update+ works pretty well.